scripts/cosm/template/.wasmd/config/genesis.json

```diff
@@ -19,18 +19,25 @@
   },
   "app_hash": "",
   "app_state": {
-    "slashing": {
+    "distribution": {
```

Comment:
Unfortunately, all the order of all those genesis keys changes for every generate. Any idea how to fix them?

Reply:
I see why it is annoying. This can't really be fixed sdk side. Best solution is to pipe this through a command that sorts the json.

This is how they compare exports in cosmos hub